src/lib.rs

```diff
@@ -72,6 +73,7 @@ mod world;
 pub use archetype::Archetype;
 pub use batch::{BatchIncomplete, BatchWriter, ColumnBatch, ColumnBatchBuilder, ColumnBatchType};
 pub use bundle::{Bundle, DynamicBundle, MissingComponent};
+pub use command_buffer::{CommandBuffer, ReadyBuffer};
```

Comment:
`ReadyBuffer` is not needed in the public API.
```suggestion
pub use command_buffer::CommandBuffer;
```